.github/workflows/SAST.yml

```
@@ -0,0 +1,64 @@
+    - name: Upload security reports
+      uses: actions/upload-artifact@v4
+      if: always()
+      with:
+        name: sast-reports
+        path: |
+          bandit-report.json
+          safety-report.json
+          semgrep-report.json
+
+# CodeQL Analysis (GitHub's built-in SAST)
+codeql-analysis:
```

actionlint (1.7.10)

[error] unexpected key "codeql-analysis" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)